Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -20,88 +20,85 @@ BatteryStateOfChargeService::~BatteryStateOfChargeService()
 
 double BatteryStateOfChargeService::totalAmpHoursUsed() const
 {
-return totalAmpHoursUsed_;
-
+    return TotalAmpHoursUsed_;
 }
 
 bool BatteryStateOfChargeService::isCharging() const
 {
-    return isCharging_;
+    return IsCharging_;
 }
 
 QTime BatteryStateOfChargeService::timeWhenChargedOrDepleted() const
 {
-   return timeWhenChargedOrDepleted_;
+   return TimeWhenChargedOrDepleted_;
 }
 
 void BatteryStateOfChargeService::addData(const BatteryData& batteryData)
 {
-    //Q_UNUSED(batteryData);
-    // This is where you can update your variables
-    // Hint: There are many different ways that the totalAmpHoursUsed can be updated
-    // i.e: Taking a running average of your data values, using most recent data points, etc.
-
-    old_current=present_batterycurrent;
-    present_batterycurrent = batteryData.current;
-    old_time=present_time;
-    present_time= batteryData.time;
+    OldCurrent_= PresentBatteryCurrent_;
+    PresentBatteryCurrent_= batteryData.current;
+    OldTime_= PresentTime_;
+    PresentTime_= batteryData.time;
```

Comment:
It may seem trivial, but the variable names should start with a lower case. Ex: OldCurrent_ -> oldCurrent_. Just to follow our standards. Im only pointing it out here, but you have to go through and fix most of your variables.